src/services/statistic/docs/index.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
 					type: 'boolean',
 				},
 			],
-			summary: 'Returns the count of a single collection according to the createdAt value of every entry as either 2 arrays or an object containing all values.',
+			summary:
+				'Returns the count of a single collection according to the createdAt value of every entry as either 2 arrays or an object containing all values.',
```

Comment:
Codacy found an issue: [This line has a length of 162. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)